### ISSUE TITLE
"Question NaN" appears in AttemptLogList in some Exercise Reports

### ISSUE THREAD

**Issue by yogeshmhaskule** [learningequality/kolibri#5247]
### Observed behavior

shows **Nan** in answer history of lesson.

### Expected behavior

Show question number in answer history instead of **Nan**

### User-facing consequences

login using coach account. And check the lesson report.

### Steps to reproduce

1. create lesson and assign to class or group.
2. solved this lesson using learner account.
3. login using coach account and check the lesson report for respective learner.

…

### Context

Tell us about your environment, including:
 * Kolibri 0.12.0
 * ubuntu 16.04
 * chrome

Note:- It show the number in front of question in the answer history, **If I refresh the page.** 
![new_issue_1](https://user-images.githubusercontent.com/7850420/53875086-cdacc700-4029-11e9-9eb8-bb7f46453eab.png)
![lesson_new_issue](https://user-images.githubusercontent.com/7850420/53875117-dbfae300-4029-11e9-99ed-da685d9d5d45.png)

**Comment by jonboiser** [comment]
I'm not able to reproduce this using the same Exercises, but I can see how this can happen if this attempt logs are not annotated with a question number at this line

https://github.com/learningequality/kolibri/blob/52371b82189ca6ccebd6f8455b5c5b76ff6735bf/kolibri/plugins/coach/assets/src/modules/exerciseDetail/actions.js#L15-L18

**Comment by jonboiser** [comment]
Don't know if this is what's causing problems, but this call to `setAttemptLogs` (the function in the snippet above) look incorrect

https://github.com/learningequality/kolibri/blob/52371b82189ca6ccebd6f8455b5c5b76ff6735bf/kolibri/plugins/coach/assets/src/views/reports/LearnerExerciseDetailPage/LearnerExerciseReportOld.vue#L136-L143

```
exercise: this.exercise // <- exercise should be a string, this.exercise looks like an object
```

**Comment by rtibbles** [comment]
This does indeed have the same root cause as #5271 - managed to replicate.

**Comment by rtibbles** [comment]
Fixed in #5298
